storybookjs/storybook
# UI: Styling updates

Issue: #14737 and other UI improvements


## What I did
- ArgsTable when in controls panel: Fix ugly black border in Safari
- Boolean control: Switch position of `True | False` to `False | True` per #14737
- Sidebar: Buttons for root heading and expand/collapse now only get the outline-y focus state if they're keyboarded to
- Toolbar icons: Update copy icon to a link (which is what it does) and share icon to the alternative share icon
- Icons: Update icon set to have parity with the design system
- a11y addon: minor layout alignment fixes

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes. Go to the PR check.
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no
